openqa-trigger-bisect-jobs

```diff
@@ -47,28 +48,60 @@ def call(cmds, dry_run=False):
 def openqa_clone(cmds, dry_run, default_opts=['--skip-chained-deps', '--within-instance'], default_cmds=['_GROUP=0']):
     return call(['openqa-clone-job'] + default_opts + cmds + default_cmds, dry_run)
 
+def fetch_url(url, request_type='text'):
+    if url.scheme == "file":
+        decodejson = lambda c: json.loads(c)
+        path = url.geturl()
+        path = path[7:]
+        path = 'test/data/python-requests/' + path
+        with open(path, "r") as request:
+            raw = content = request.read()
+    elif url.scheme in ["http", "https"]:
+        decodejson = lambda c: content.json()
+        try:
+            content = requests.get(url.geturl())
+        except requests.exceptions.RequestException as e:
+            log.error("Error while fetching "+url.geturl()+": "+str(e))
+            raise(e)
```

Comment:
why catch these cases at all when you don't handle them except for logging?

Reply:
Because the exception itself does not print the url and/or content it failed for?
Remember the problems we had with the gru logs, having failures where we neither had the url no the content?
Do we want  to have the same problems back after we add openqa-trigger-bisect-jobs or should I rather add the exception handling **now**?